### PR TITLE
pdpb:add approximate_count for RegionHeartbeatRequest

### PR DESCRIPTION
**Pull request by AndreMouche**
@BusyJay @disksing PTAL

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -242,7 +242,8 @@ message RegionHeartbeatRequest {
     uint64 keys_read = 9;
     // Approximate region size.
     uint64 approximate_size = 10;
-    reserved 11;
+    // Approximate number of keys.
+    uint64 approximate_count = 11;
```
Comment (disksing) [inline proto/pdpb.proto]: 11 is reserved for backward compatibility. You can use 13 instead.

Comment (siddontang) [reply proto/pdpb.proto]: not 12?

Comment (AndreMouche) [reply proto/pdpb.proto]: 12 has already been used for field `interval`.

File: proto/pdpb.proto [hunk 2]
```diff
@@ -245,6 +245,8 @@ message RegionHeartbeatRequest {
     reserved 11;
     // Actually reported time interval
     TimeInterval interval = 12;
+    // Approximate number of keys.
+    uint64 approximate_count = 13;
```
Comment (overvenus) [inline proto/pdpb.proto]: approximate_key_count

Comment (AndreMouche) [reply proto/pdpb.proto]: maybe `approximate_rows`?

**Pull request merged**